docs/build_node.md

```diff
@@ -378,14 +378,22 @@ while (true) {
 
 **Dependencies:** `ChannelManager`, `ChainMonitor`
 
-### Once Per Minute: `ChannelManager`'s `timer_chan_freshness_every_min()`
-**What it's used for:** `ChannelManager` needs to be told every time a minute passes so that it can broadcast fresh channel updates if needed
+### Background Processing
+
+`ChannelManager`'s `timer_chan_freshness_every_min()`
+
+**What it's used for:** running tasks periodically that aren't high-priority, e.g.,
```

Comment:
Could we take out the `e.g.` -- `.e.g.` means "for example," and those two steps aren't examples, they're the entire step.